src/tests/ftest/cart/util/cart_logparse.py

```diff
@@ -404,9 +405,10 @@ def free_pointer(self):
         """Return the memory address freed"""
         return self.get_field(-1).rstrip('.')
 
+
 # pylint: disable=too-many-branches
 class StateIter():
-    """Helper class for LogIter to add a statefull iterator.
+    """Helper class for LogIter to add a state-full iterator.
```

Comment:
Should this actually be `stateful`? Probably isn't in the dictionary, but I think that's the proper term here